actionpack/lib/action_controller/metal/streaming.rb

```diff
@@ -188,7 +188,14 @@ module ActionController #:nodoc:
   #
   # ==== Passenger
   #
-  # To be described.
+  # Phusion Passenger with NGINX, offers two streaming mechanisms out of the box. NGINX response
```

Comment:
Yeah the paragraphs below this are a little weird. They also tend to run long and need some more space between them.

I might be able to offer some examples shortly, but yeah this one could definitely use some help.